src/main/java/seedu/duke/data/Module.java

```diff
@@ -72,13 +73,16 @@ public void setTaskArrayList(ArrayList<Task> list) {
 
     /**
      * Adds one task in task list associated with the module.
+     * @param task The task to be added
      */
     public void addTask(Task task) {
         taskList.addTask(task);
     }
 
     /**
      * Formats the module and its associated tasks according to given parameters.
+     * @param showCompletedTasks Whether completed tasks should be shown or not
+     * @return The string of all tasks, depending on whether completed tasks should be shown
      */
     public String printModuleTaskList(boolean showCompletedTasks) {
         return this + LS + taskList.getAllTasks(INDENT, showCompletedTasks);
```

Comment:
Suggests to modify the method name to getModuleTaskList or moduleTaskListToString, or something similar, because I don't think it prints here.

Reply:
This is not for this PR specificly, but just a small suggestion.

Reply:
renamed to `stringifyModuleTaskList`